### DISCUSSION TITLE
rate limits for free tier openai - gpt prompt not returning

### DISCUSSION THREAD

**Discussion by fr8nki3** [Kav-K/GPTDiscord#439] [Q&A] [answered]
Looks like this is an openai user issue with tiers maybe, I am new and just working this out. I am in the free tier, how do I modify to use gpt?

![image](https://github.com/Kav-K/GPTDiscord/assets/56416248/2cdd4bf5-6046-4a6a-8a7e-87f02c482cd3)

![image](https://github.com/Kav-K/GPTDiscord/assets/56416248/1e27bf36-1b62-40bd-a3f3-d265740eabbf)

**Comment by Kav-K** [comment, marked as answer]
You need to put money into the API account and get out of free tier to use the bot properly unfortunately
